### PR TITLE
1655 fixed missing refresh after delete

### LINKED ISSUE #1655: Easy Editor: Fix annoying display bug
After deleting or inserting nodes in the easy editor, the parent node needs to be closed and re-opened for the display to update correctly.

https://discord.com/channels/717692382849663036/808086068023918603/1232949549765758998

### PR DESCRIPTION
**Pull request by manavortex**
closes #1655

### CODE REVIEW

File: WolvenKit.App/ViewModels/Tools/ChunkViewModel.UserInteractionStates.cs
Comment (dragonzkiller) [inline WolvenKit.App/ViewModels/Tools/ChunkViewModel.UserInteractionStates.cs:1]: what is this change doing? it doesn't seem to be part of the issue this PR resolves

Comment (manavortex) [reply WolvenKit.App/ViewModels/Tools/ChunkViewModel.UserInteractionStates.cs:1]: making buffers writable again. it's unrelated, but it's a small change, and it'll resolve an issue that sim has

Comment (dragonzkiller) [reply WolvenKit.App/ViewModels/Tools/ChunkViewModel.UserInteractionStates.cs:1]: I don't really like unrelated changes especially in PRs that are dedicated to closing an issue. But I'll approve it this time

**Review by dragonzkiller — CHANGES_REQUESTED** [review]
Resolve the conflict and I can approve

**Review by dragonzkiller — APPROVED** [review]

**Pull request merged**